src/common/cuepoint/cuepoint-manager.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import {Cue, FakeEvent, TextTrack} from '@playkit-js/playkit-js';
+import {CUE_POINTS_TEXT_TRACK, CUE_POINT_KEY} from './cuepoint-type';
+import {CuePointEventType} from './cuepoint-event-type';
+
+interface CuePoint {
+  id: string;
+  startTime: number;
+  endTime: number;
+}
+
+export class CuePointManager {
```

Comment:
@semarche-kaltura any reason for defining all methods as arrow function? this is a class and it is instantiated so `this` is always the correct context

Reply:
changed